packages/connect-form/src/components/connect-form-actions.tsx

```diff
@@ -11,37 +11,76 @@ import {
   ConnectionFormError,
   ConnectionFormWarning,
 } from '../utils/validation';
+import type { ConnectionInfo } from 'mongodb-data-service';
 
 const formActionStyles = css({
   borderTop: `1px solid ${uiColors.gray.light2}`,
   paddingLeft: spacing[4],
   paddingRight: spacing[4],
 });
 
+const formActionButtonStyles = css({
```

Comment:
suggestion: this is a container of multiple buttons and the name of the component is "form-actions", so maybe for consistency could be better to use the plural here for both:

```suggestion
const formActionsButtonsStyles = css({
```